Client/src/Pages/NotFound/index.css

```diff
@@ -1 +1,35 @@
-/*Notfoundpage styles*/
\ No newline at end of file
+/*Notfoundpage styles*/
+.not-found-page,
+.not-found-template,
+#not-found-title,
+#not-found-description {
+  justify-content: center;
+  align-items: center;
+}
+
+.not-found-page {
+  display: flex;
+  min-height: 100vh;
+}
+
+.not-found-template {
+  min-height: 170px;
+}
+
+#not-found-title,
+#not-found-description {
+  text-align: center;
+}
+
+#not-found-title {
+  font-size: var(--env-var-font-size-large);
+  color: var(--env-var-color-5);
+  font-weight: 600;
+  margin-bottom: var(--env-var-spacing-2);
+}
+
+#not-found-description {
+  font-size: var(--env-var-font-size-medium);
+  font-weight: 500;
+  margin-bottom: calc(var(--env-var-spacing-4) + var(--env-var-spacing-2));
+}
```

Comment:
It seems to me a good amount of this CSS isn't actually needed if we simplify our structure a bit:

Imagine this simplified format:

```
    <Stack className="not-found-page" justifyContent="center">
      <Stack gap="20px" alignItems="center">
        <Typography component="h1">{title}</Typography>
        <Typography>{desc}</Typography>
        <Button
          label="Go to the main dashboard"
          level="primary"
          sx={{ mt: "24px" }}
        />
      </Stack>
    </Stack>

```
    
It would seem to me the only css we need is:

```
.not-found-page {
  height: 100vh;
}
.not-found-page button,
.not-found-page p {
  font-size: var(--env-var-font-size-medium);
}
.not-found-page h1 {
  font-size: var(--env-var-font-size-large);
  font-weight: 600;
}
.not-found-page h1,
.not-found-page p {
  color: var(--env-var-color-5);
}
```

Can you test out this implementation ad see what you think?

We can leverage the properties of CSS specificity to target elements without IDs and target multiple elements at the same time.

This will help keep our codebase maintainable, reduce cognitive load, and reduce the size of our bundled code which leads to better performance of our app!

Try it out and see what you think!

Reply:
Yes, Master! 😂✌️🙏🏻🙏🏻
This turned out exactly like you said!

![HowPaapabutterflyGIF](https://github.com/user-attachments/assets/48c2dae8-bf49-4601-89c5-17470c4c3c62)